canonical-web-and-design/maas-ui
# fix(legacy): fix tooltips in the controllers table

## Done

- Update how the controllers are truncated in the table so that the tooltips are not cut off.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- You'll probably need a MAAS with sampledata to get controllers with different versions.
- Visit the controller list.
- In the version column hover an (i) icon. You should see the tooltip appear.

## Fixes

Fixes: #2487.
